docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,366 @@
+- More TBD.
+
+Open Questions: 
+- Is a resolved registry contain any trace of the imported registries?
+- It will be nice to detect when a new registry version is available. How can we detect the presence of a new registry
+  version? A file containing of the schema URL per version for a registry? Some kind of naming conventions for the URL? 
+
+## Protocol Changes
+
+No impact on OTLP and OTAP. 
+
+A `schema_url` field is already present at the resource and scope levels. 
+
+Ideally any component of the observability pipeline should be able to fetch the resolved registry just by knowing the
+schema URL of any resource or instrumentation scope. 
+
+## OTEL SDKs Changes
+
+TBD
+
+Open Questions:
+
+- Can we enforce the presence of the schema URL in the resource and instrumentation scope?
+- How do we convey the schema URL to the SDKs? Could that be part of the codegen done by Weaver?
+
+## Resolved Semantic Convention Registry Format
+
+> Note: A resolved registry is self-contained and does not include any complex constructs
+> like `imports`, `ref`, `extends`, etc. Their **structure is less subject to change**, making them good
+> candidates for publication and easier to consume.
+
+The following properties are proposed for a resolved registry:
+
+- Resolved registry must be easy to consume and to publish
+  - Accessible via a URL.
+  - Self-contained, i.e. a single file.
+  - No `ref`, no `extends`, no `imports`, no alias, no other complex constructs.
+  - Yaml or JSON format so resolved registries can be easily consumed by any tool.
+- Optional lineage section.
+
+The structure of resolved registries is less subject to change. We can always add advanced support for embedding,
+more complex inheritance mechanisms, etc. The resolved registry format will not change because all these complex
+constructs will be removed during the resolution process. This property makes resolved registries good candidates
+for publication and easier to consume.
+
+The content of a resolved registry depends on the:
+- The semantic convention files composing the registry to resolve.
+- The registries imported.
+- The configuration specified during the resolution process.
+  - Include all the entities of the imported registries
+  - Include only the referenced entities of the imported registries.
+
+More specifically, a resolved registry contains:
+- All the attributes registry specified locally in the registry.
+- All the groups specified locally in the registry.
+- All the attributes and groups imported but not re-exported locally are not included in the resolved registry. A
+  re-exported entity is an entity that is imported and referenced in the local registry with some overriding.
+
+Open Questions:
+
+- Do we keep track of the imported registries in the resolved registry? If yes, how? Lineage?
+- Can we leverage the attribute deduplication mechanism to simplify the merging of imported registries? ToDo -> Explain
```

Comment:
can we start by disallowing attribute/span/event/metric duplication and only add it along with scoping only if there is actual demand?

Reply:
The next version of this document includes a complete revision of the resolution process and eliminates the need for scoping in the most common cases. I think the best approach at this stage is to postpone the discussion on scoping.